File: OneCodeTeam/Story-Creator-Sample-Application-for-Windows-14638/[C#][C++]-Story Creator Sample Application for Windows Phone 7 (CSWP7AzureVideoStory)/C# and C++/C#,C++/CSWP7AzureVideoStory/VideoStoryCreator/ViewModels/ChoosePhotoViewModel.cs
/****************************** Module Header ******************************\
* Module Name:	ChoosePhotoViewModel.cs
* Project:		VideoStoryCreator
* Copyright (c) Microsoft Corporation.
* 
* A view model used in the choose photo page.
* 
* This source is subject to the Microsoft Public License.
* See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
* All other rights reserved.
* 
* THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
* EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
* WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/


namespace VideoStoryCreator.ViewModels
{
    public class ChoosePhotoViewModel : PhotoViewModel
    {
        public bool IsSelected { get; set; }
    }
}

File: OneCodeTeam/Story-Creator-Sample-Application-for-Windows-14992/[VB][C++]-Story Creator Sample Application for Windows Phone 7 (VBWPAzureVideoStory)/C++ and VB/C++,VB/VBWPAzureVideoStory/VideoStoryCreator/ViewModels/ChoosePhotoViewModel.vb
'********************************* Module Header *********************************\
' Module Name: ChoosePhotoViewModel.vb
' Project: VideoStoryCreator
' Copyright (c) Microsoft Corporation.
' 
' A view model used in the choose photo page.
' 
' This source is subject to the Microsoft Public License.
' See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
' All other rights reserved.
' 
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
' EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
' WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'***************************************************************************/



Public Class ChoosePhotoViewModel
    Inherits PhotoViewModel

    Public Property IsSelected() As Boolean
        Get
            Return m_IsSelected
        End Get
        Set(value As Boolean)
            m_IsSelected = Value
        End Set
    End Property
    Private m_IsSelected As Boolean
End Class
